source/components/molecules/CaseCard/CaseCard.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Icon, Text } from '../../atoms';
+import { Card } from '..';
+
+Card.LargeText = styled(Card.Text)`
+  font-size: ${(props) => props.theme.fontSizes[4]}px;
+  font-weight: ${(props) => props.theme.fontWeights[1]};
+  margin-bottom: 4px;
+`;
+
+Card.Meta = styled(Card.Text)`
+  font-size: ${(props) => props.theme.fontSizes[3]}px;
+  ${(props) => `color: ${props.theme.colors.neutrals[1]};`}
+`;
+
+const CaseCard = ({
+  colorSchema,
+  icon,
+  name,
+  largeSubtitle,
+  subtitle,
+  description,
+  onCardClick,
+  showProgress,
+  currentStep,
+  totalSteps,
+  showButton,
+  buttonText,
+  buttonIconName,
+  onButtonClick,
+  showPayments,
+  approvedAmount,
+  declinedAmount,
+  givedate,
+}) => (
+  <Card colorSchema={colorSchema}>
+    <Card.Body shadow color="neutral" onPress={onCardClick}>
+      {icon && <Card.Image source={icon} />}
+      <Card.Title colorSchema="neutral">{name}</Card.Title>
+      {largeSubtitle && <Card.LargeText mt={0.5}>{largeSubtitle}</Card.LargeText>}
```

Comment:
What's the different between the subtitle and the largeSubtitle in relation to the figma boards :)
Can't find any use cases where we have a large subTitle directly underneath the title?

Reply:
This structure is just based on the previous code that this is refactored from, which doesn't seem to map 1:1 to the Figma design, unless my refactor is inaccurate.

The picture below is pre-refactor, where "Juli" is `largeSubtitle` and "Inskickad" is the regular `subtitle`.

![image](https://user-images.githubusercontent.com/81566071/126281028-89b4aba1-93ec-483a-9b8d-727573afbf92.png)

In my opinion we should keep this as-is for now and then later do a "fix design" PR since there are a few more inconsistencies throughout the case screens outside this scope.

Reply:
Sounds like a reasonable idea in order to move forward :)

Reply:
Can we document the inconsistencies somewhere? So that they don't get burried or forgotten during the process :)

Reply:
I can make a card of it in ClickUp 👍.